### PR TITLE
Fixed artipie config parsing

### PR DESCRIPTION
**Pull request by g4s8**
#33 - fixed Artipie server and config parsing

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#34](https://github.com/artipie/artipie/issues/34) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/50032728-e58b-4735-a85e-60ee2d5c269a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/Pie.java [hunk 1]
```diff
@@ -67,20 +70,24 @@ public Pie(final Settings settings) {
     @SuppressWarnings("PMD.OnlyOneReturn")
     public Response response(final String line, final Iterable<Map.Entry<String, String>> headers,
         final Publisher<ByteBuffer> body) {
+        Logger.info(this, "Request: %s", line);
         final URI uri = new RequestLineFrom(line).uri();
         if (uri.getPath().equals("*")) {
             return new RsWithStatus(200);
         }
-        final String[] path = uri.getPath().split("/");
+        final String[] path = uri.getPath().replaceAll("^/+", "").split("/");
```
Comment (fanifieiev) [inline src/main/java/com/artipie/Pie.java]: @g4s8 Would be smarter to assign `uri.getPath()` to a variable once after line 74  and use it.
This method `response` defines `uri.getPath()` twice.

File: pom.xml [hunk 2]
```diff
@@ -121,6 +121,16 @@ SOFTWARE.
       <artifactId>files-adapter</artifactId>
       <version>0.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
```
Comment (fanifieiev) [inline pom.xml]: @g4s8 As far as I understand, you importing log4j1.2 into the project, but I heard there are some issues with that in Java 9. 
Please take a look at https://blogs.apache.org/logging/entry/moving_on_to_log4j_2

File: src/main/java/com/artipie/Pie.java (hunk 1 above)
Comment (g4s8) [reply src/main/java/com/artipie/Pie.java]: @fanifieiev right, fixed

File: pom.xml (hunk 2 above)
Comment (g4s8) [reply pom.xml]: @fanifieiev I checked it on JDK13 and it's working fine, you may want to submit a ticket with proposal for update, but I don't think we need to do it right now

**Comment by 0crat** [comment]
@fanifieiev[/z](https://www.0crat.com/u/fanifieiev) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @fanifieiev[/z](https://www.0crat.com/u/fanifieiev)[/z](https://www.0crat.com/u/fanifieiev) didn't make CR comments

**Comment by 0crat** [comment]
Order was finished: +15 point(s) just awarded to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**